src/main.html

```diff
@@ -71,21 +70,21 @@
     <script type="text/javascript" src="./node_modules/short-unique-id/dist/short-unique-id.min.js"></script>
     <script type="text/javascript" src="./node_modules/object-hash/dist/object_hash.js"></script>
     <script type="text/javascript" src="./js/libraries/q.js"></script>
-    <script type="text/javascript" src="./js/libraries/jquery-2.1.4.min.js"></script>
+    <script type="text/javascript" src="./node_modules/jquery/dist/jquery.min.js"></script>
```

Comment:
I'd probably stick the `./node_modules` sources up top, where the other ones are.